constellation_base/views.py

```diff
@@ -57,3 +62,52 @@ def login_view(request):
 def logout_view(request):
     logout(request)
     return redirect("Login")
+
+
+@method_decorator(staff_member_required, name='dispatch')
+class manage_groups(View):
+
+    def get(self, request):
+        """Show a list of the supplemental groups"""
+        template_settings_object = GlobalTemplateSettings(allowBackground=True)
+        template_settings = template_settings_object.settings_dict()
+
+        existing_groups = LocalGroupACLEntry.objects.values_list('group',
+                                                                 flat=True)
+        existing_groups = Group.objects.filter(pk__in=set(existing_groups))
```

Comment:
Set is used to compute unique values and then only pull one of each group, since there are potentially many individual entries in the table.